README.md

```diff
@@ -40,7 +40,7 @@
     
 | TAIPY GUI - the frond-end  | TAIPY Core - the back-end |
 | --------  | -------- |
-|<img src="readme_img/readme_gui_intro.gif" alt="Taipy Logo"  width="350" height="300"/> | <img src="readme_img/readme_core_intro.gif" alt="Taipy Logo"  width="350" height="300"/>
+|<img src="readme_img/readme_gui_intro.gif" alt="Taipy GUI Gif"  width="100%"/> | <img src="readme_img/readme_core_intro.gif" alt="Taipy Core Gif"  width="100%"/>
```

Comment:
You should not specify the file type in the alt entry. That should be a description of the content, not how it is done.
=> "Animated image for introducing Taipy Core "

Reply:
Fixed!